src/main/java/org/fundacionjala/coding/maria/CifradoCesar.java

```diff
@@ -0,0 +1,136 @@
+package org.fundacionjala.coding.maria;
+
+
+/**
+ * kata cifrado cesar.
+ */
+public class CifradoCesar {
```

Comment:
Split in two classes Cesar and Vigenere and apply abstraction for encode and decode